### PR TITLE
For Code review

### PR DESCRIPTION
**Pull request by Itswali**
`Live Video Link` - https://www.loom.com/share/99cd407a2c5a4bb89e00d6b13eb50f9d

- [x] Use React documentation.
- [x] Use React components.
- [x] Use React props.
- [x] Use React Router.
- [x] Connect React and Redux.
- [x] Handle events in a React app.
- [x] Write integration tests with a React testing library.
- [x] Use styles in a React app.
- [x] Use React life cycle methods.
- [x] Apply React best practices and language style guides in code.
- [x] Use store, actions and reducers in React.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -133,9 +130,9 @@ To to build for the production, execute the following command:
...
 
 
-- [ ] **............**
-- [ ] **............**
-- [ ] **............**
+- [ ] **User can search country by code**
+- [ ] **User can compare country by population**
+- [ ] **User can see population increase ration of countries**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (Mihndim2020) [inline README.md:141]: - [ ] As it is a [requirement ](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md)for the project, "`
The Creative Commons license of the design requires that you give appropriate credit to the author. Therefore, you must do it in the README of your project.`", please in your `Acknowledgements Section` give appropriate credit to [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio) for his original design idea.

File: src/tests/Details.test.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    const wrapper = render(
+      <Provider store={store}>
+        <MemoryRouter>
+          <Details />
+        </MemoryRouter>
+      </Provider>,
+    );
+    expect(wrapper).toMatchSnapshot();
+  });
+});
```
Comment (Mihndim2020) [inline src/tests/Details.test.js:21]: - [ ] Good job with test coverage but some of your tests are failing, please kindly checkout the screenshot below and fix the issue. 

failingTest

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @Itswali, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Clean and Organized code
:heavy_check_mark: Good job with UI
:heavy_check_mark: Good job deploying the project

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION
# Status: Requires Changes ♻️
_Check the comments below and under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
Hi @Itswali, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

> ## (Highlights) Good Points: 💪 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:

çrpik-kurdî-impressive



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear
Please, remember to tag @topeogunleye in your question so I can receive the notification.
 
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**